Test failure: ImportError: libcuda.so.1

### Summary

Tests are failing on github actions, probably since merging PR #958 

Could be because we now have cupy installed, but the nodes do not have a working CUDA setup.

Eg.
https://github.com/mantidproject/mantidimaging/runs/2519777397?check_suite_focus=true


### Failure Logs

```
 ___________ ERROR collecting mantidimaging/core/gpu/test/gpu_test.py ___________
ImportError while importing test module '/home/runner/work/mantidimaging/mantidimaging/mantidimaging/core/gpu/test/gpu_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/share/miniconda/envs/mantidimaging-dev/lib/python3.8/site-packages/cupy/__init__.py:16: in <module>
    from cupy import _core  # NOQA
/usr/share/miniconda/envs/mantidimaging-dev/lib/python3.8/site-packages/cupy/_core/__init__.py:1: in <module>
    from cupy._core import core  # NOQA
cupy/_core/core.pyx:1: in init cupy._core.core
    ???
/usr/share/miniconda/envs/mantidimaging-dev/lib/python3.8/site-packages/cupy/cuda/__init__.py:8: in <module>
    from cupy.cuda import compiler  # NOQA
/usr/share/miniconda/envs/mantidimaging-dev/lib/python3.8/site-packages/cupy/cuda/compiler.py:12: in <module>
    from cupy.cuda import function
cupy/cuda/function.pyx:1: in init cupy.cuda.function
    ???
cupy/cuda/texture.pyx:1: in init cupy.cuda.texture
    ???
E   ImportError: libcuda.so.1: cannot open shared object file: No such file or directory

The above exception was the direct cause of the following exception:
/usr/share/miniconda/envs/mantidimaging-dev/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
mantidimaging/core/gpu/test/gpu_test.py:10: in <module>
    from mantidimaging.core.operations.median_filter import MedianFilter, modes
mantidimaging/core/operations/median_filter/__init__.py:4: in <module>
    from .median_filter import MedianFilter, modes  # noqa:F401
mantidimaging/core/operations/median_filter/median_filter.py:12: in <module>
    from mantidimaging.core.gpu import utility as gpu
mantidimaging/core/gpu/utility.py:12: in <module>
    import cupy as cp
/usr/share/miniconda/envs/mantidimaging-dev/lib/python3.8/site-packages/cupy/__init__.py:37: in <module>
    raise ImportError(_msg) from e
E   ImportError: CuPy is not correctly installed.
E   
E   If you are using wheel distribution (cupy-cudaXX), make sure that the version of CuPy you installed matches with the version of CUDA on your host.
E   Also, confirm that only one CuPy package is installed:
E     $ pip freeze
E   
E   If you are building CuPy from source, please check your environment, uninstall CuPy and reinstall it with:
E     $ pip install cupy --no-cache-dir -vvvv
E   
E   Check the Installation Guide for details:
E     https://docs.cupy.dev/en/latest/install.html
E   
E   original error: libcuda.so.1: cannot open shared object file: No such file or directory
```

--- a/docs/release_notes/next.rst
+++ b/docs/release_notes/next.rst
@@ -15,6 +15,7 @@ Fixes
 
 - #939 : median: NameError: name 'cp' is not defined
 - #957 : Crop Coordinates ROI not changing
+- #961 :  Test failure: ImportError: libcuda.so.1
 
 Developer Changes
 -----------------

--- a/mantidimaging/core/gpu/utility.py
+++ b/mantidimaging/core/gpu/utility.py
@@ -12,6 +12,10 @@ try:
     import cupy as cp
     mempool = cp.get_default_memory_pool()
 except ModuleNotFoundError:
+    # cupy not installed
+    CUPY_NOT_IMPORTED = True
+except ImportError:
+    # cupy installed, but unable to load CUDA
     CUPY_NOT_IMPORTED = True
 
 MAX_CUPY_MEMORY_FRACTION = 0.8

